client/signals/get_v1_signals_ingest_url_responses.go

```diff
@@ -39,9 +42,10 @@ func NewGetV1SignalsIngestURLOK() *GetV1SignalsIngestURLOK {
 /*
 GetV1SignalsIngestURLOK describes a response with status code 200, with default header values.
 
-get IngestURL(s)
+Retrieve the url for ingesting signals for your organization
 */
 type GetV1SignalsIngestURLOK struct {
+	Payload *models.SignalsAPIIngestKeyEntity
```

Comment:
Heyo looky there!

Reply:
is this what you were working on? 🤣

Reply:
ya, most of our signals endpoints don't properly infer what the response will be for get requests because of how we don't add swagger `desc` https://github.com/firehydrant/laddertruck/pull/10752/files

Reply:
this is the case throughout a lot of the app. we've fixed some of it over time, but i'm sure there's more.